src/modals/vm-export/VmExportDataProvider.js

```diff
@@ -40,10 +40,16 @@ async function validateThinVm (vmId, sdId) {
   const vm = await engineGet(`api/vms/${vmId}`)
 
   // If a VM has both attributes set it's thin/dependent
-  if (vm.original_template.id === emptyGuid || vm.template.id === emptyGuid) {
+  if (vm.template.id === emptyGuid) {
     return true
   }
 
+  if (!vm.original_template.id == null) {
```

Comment:
I am not that experienced with js, but this looks like negating only the first term (?).

Maybe better `vm.original_template.id !== null`?
Or if I'm not mistaken, this should work too: `if (vm.original_template.id)` to check if exists.

Also, I feel it would be better to keep these nested conditions together with a `&&`.

Reply:
If `vm.original_template` is what could be `null`, this change will not work.  At runtime you'll probably get a 'Cannot read properties of undefined' error message since it is trying to read `id` from `null`.

The best change is what @mkemel suggested above.  Just use optional chaining (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) and then the result of `vm.original_template` being null will produce an `undefined` instead of an error.

Some simple examples:
```
# node
Welcome to Node.js v16.14.0.
Type ".help" for more information.
> let a = { b: { id: 'wednesday' }, c: null };
undefined
> a.b.id
'wednesday'
> a.b.id2
undefined
> !a.b.id
false
> !a.b.id2
true
> a.c.id
Uncaught TypeError: Cannot read properties of null (reading 'id')
> a.c?.id
undefined
```